### ISSUE TITLE
Properly Integrate TailwindCSS into Laravel

### ISSUE THREAD

**Issue by faramafu** [nurfachmi/perpustakaan#18]
## Issue Description:
I've identified an opportunity to enhance this project by properly integrating Tailwind CSS into it. Currently, Tailwind CSS is inserted manually, and Blade templates are used for style reuse. However, I'd like to help improve this setup by integrating TailwindCSS with the Vite builder to enable tree shaking.

## Status:

Current: 
Tailwind CSS inserted manually, using Blade templates for reuse.
Plan: 
Integrate with Vite builder, enable tree shaking.

## Expected Behavior:
TailwindCSS should be properly integrated into the project, with the ability to leverage Vite for more efficient asset management and tree shaking.

## Additional Context:
I believe this enhancement will not only streamline the development process for THIS project but also improve performance by eliminating unused CSS from the final build. Looking forward to discussing the implementation details and potential challenges.

**Comment by akbarhlubis** [comment]
Agreed, selain lebih optimize size css nya, dari segi design juga bisa lebih mudah di customize dibanding Bootstrap. 

Mungkin challenge nya lebih ke ekstra pembuatan komponen responsive nya yang kalau dibanding Bootstrap yang biasanya sudah include responsif dan animation nya.

Solusinya kalau biar cepat bisa pakai Alpine.js maybe(?)

**Comment by AaEzha** [comment]
saya gak ada masalah untuk styling. saat ini pakai Bootstrap karena paling sederhana dan memang yang saya mampu. Justru ingin belajar TailwindCSS sekalian nyemplung ke projek real.
